data/json/snippets/survivor_notes.json

```diff
@@ -687,6 +687,169 @@
+      },
+      {
+        "id": "msn_note_23",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"I knew I shoulda killed that spider in the basement.  Now there is a mob of giant spiders.  Fuck this place.\""
+      },
+      {
+        "id": "msn_note_24",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Do not go in the fucking basement my dude.\""
+      },
+      {
+        "id": "msn_note_25",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"Call me at: 0406 650 430\""
+      },
+      {
+        "id": "msn_note_26",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"Fuck, I don't think this bug spray is going to work.\""
+      },
+      {
+        "id": "msn_note_27",
+        "//": "Thanks to 'TooFewSecrets' for helping me by providing this snippet idea.",
+        "text": "\"Those spores don't burn, but the rest of the mushrooms sure as Hell do.\""
+      },
+      {
+        "id": "msn_note_28",
+        "//": "Thanks to Aeron for helping me by providing this snippet idea.",
+        "text": "\"I saw some naked guy try to fistfight a hulk the other day.  He died, but someone who looked EXACTLY like him came by 5 minutes later attempting the same exact thing.  What the hell's going on there?\""
+      },
+      {
+        "id": "msn_note_29",
+        "//": "Thanks to Vetulus for helping me by providing this snippet idea.",
+        "text": "\"If by all laws of aviation a normal bee shouldn’t be able to fly, then how the fuck does a giant one?\""
+      },
+      {
+        "id": "msn_note_30",
+        "//": "Thanks to Lorith for helping me by providing this snippet idea.",
+        "text": "\"There was a turret on the corner yesterday.  I watched fifty people who looked the same run right up to it yelling \"SHOOT ME BITCH!\" before the last one punched it and died when it exploded.  Not five minutes later, another ran through the intersection like he knew it was safe.\""
+      },
+      {
+        "id": "msn_note_31",
+        "//": "Thanks to 'DracoGriffen' for helping me by providing this snippet idea.",
+        "text": "\"I keep having the same dream over and over: Here I am slaughtering this electrified zombie, heaps of flesh gouged out, putrified goo and parts filling my hands, slopping against my clothes and face.  I can feel the sweat pouring down my cheeks and my back, and every time I pull out some sort of burnt out bionic.  Sometimes I find a battery, or if I'm really lucky, I'll find some really extravagant gizmo and I wake up from the dream.\""
+      },
+      { "id": "msn_note_32", "text": "\"My mama always used to says \"stupid is as stupid does.\"\"" },
+      {
+        "id": "msn_note_33",
```

Comment:
Five notes in a row about Forest gump seems too much

Reply:
How does four sound?